articles/model-anatomy.adoc

```diff
@@ -0,0 +1,644 @@
+With node `34`, we have an annotation instance of `SourcePosition`.
+Node `1:Function` can be _annotated by_ node `34:SourcePosition`, because `SourcePosition` states it _annotates_ `Function`.
+In contrast, node `3:Argument` cannot be _annotated by_ node `35:SourcePosition`, because `Argument` is not compatible with `Function` (the kind of classifier ``SourcePosition`` _annotates_).
+
+[plantuml,annotation,svg]
+----
+hide empty members
+
+package M2 <<Rectangle>> {
+  class Function <<Classifier>> {
+    name: String
+  }
+
+  class Argument <<Classifier>> {
+  }
+  Function *- "0..*" Argument: parameters
+
+  annotation SourcePosition <<Annotation>> {
+    line: Integer
+    column: Integer
+  }
+  SourcePosition .> Function: annotates
+}
+
+package M1 <<Rectangle>> {
+  class f1 as "1:Function" <<Node>> {
+    name = "doIt"
+  }
+
+  class a1 as "3:Argument" <<Node>> {
+    name = "action"
+  }
+  f1 *- a1: parameters
+
+  annotation an1 as "34:SourcePosition" <<Node>> {
+    line = 2
+    column = 13
+  }
+  an1 -* f1: <<annotations>>
+
+  annotation an2 as "35:SourcePosition" <<Node>>  #line:red {
+    line = 2
+    column = 18
+  }
+  a1 *- an2: <<annotations>>
+}
+
+M2 -[hidden]- M1
+
+'Function <.. f1: <<classifier>>
+'Argument <.. a1: <<classifier>>
+'SourcePosition <.. an1: <<classifier>>
+'SourcePosition <. an2: <<classifier>>
+
+----
+
+An annotation that is applicable to all concepts says it annotates `Node` from LionWeb's built-in library.
+Annotations can _annotate_ other annotations; however, this approach introduces complexities and should be used with great caution.
+
+#TODO:# Do we allow unresolvable annotations?
+
+=== Differences between containment and annotation instance
+Both a contained node `C` and an annotated node `A` share the lifecycle of their parent node `P`.
+So what's the difference?
+
+A containment must be described in the parent's node classifier.
+If the parent is of classifier `House` with containments `rooms`, and `doors`, we cannot add a containment `windows` to the parent node.
+The designer of classifier `House` made the choice that for their domain, a house should only care about rooms and doors, but not windows.
+The designer might not have anticipated our use case of their domain.
+But every user of that classifier relies on this structure of a house, so we must not mess with it.
+
+An annotation instance cannot be mentioned in the parent's node classifier at all.
+Instead, an annotation instance's classifier must be an annotation, which mentions the classifier it annotates.
+Thus, the designer of the annotation decides which targets their annotation is applicable to.
+So we can come up with an annotation `WindowAnnotation` that `annotates: House`, and attach instances to nodes of classifier `House`.
+All these nodes still adhere to the intents of ``House``'s designer, but also transport the additional information about windows we need for our use case.
```

Comment:
Do orthogonal nodes adhere to the original intents, or rather do not mess up with them?

Reply:
Is this question answered in the next paragraph?

Reply:
What is misleading here is `All these nodes still adhere to the intents of House's designer`. I guess you mean that these new (window) nodes do not interfere with the intents of House's designer, but also transport the additional information. "Not interfering" and "adhering" are not the same here.
Imho, the house example is confusing for the introduction of annotations. So that you need to write an extra excuse for it below.